data-prepper-plugins/otel-metrics-raw-processor/src/main/java/com/amazon/dataprepper/plugins/processor/otelmetrics/OTelMetricsRawProcessor.java

```diff
@@ -222,6 +227,14 @@ private List<? extends Record<? extends Metric>> mapHistogram(final io.opentelem
 
     private List<? extends Record<? extends Metric>> mapExponentialHistogram(io.opentelemetry.proto.metrics.v1.Metric metric, String serviceName, Map<String, Object> ils, Map<String, Object> resourceAttributes, String schemaUrl) {
         return metric.getExponentialHistogram().getDataPointsList().stream()
+                .filter(dp -> {
+                    if (otelMetricsRawProcessorConfig.getExponentialHistogramMaxAllowedScale() < Math.abs(dp.getScale())){
```

Comment:
Should this allow any scale to pass through if the `calculateExponentialHistogramBuckets` is set to false? The scale is a problem only when performing the processing, right?

This might be more appropriate:

```
if (otelMetricsRawProcessorConfig.getCalculateExponentialHistogramBuckets() &&
                            otelMetricsRawProcessorConfig.getExponentialHistogramMaxAllowedScale() < Math.abs(dp.getScale())){
```

What do you think?